tutorial/README.md

```diff
@@ -0,0 +1,31 @@
+# .NET Core Diagnostics Overview
+
+With .NET Full running on Windows we have grown accustomed to a plethora of great diagnostics tools ranging from dump generation and manual analysis to more sophisticated collection engines such as DebugDiag. As .NET core is picking up (cross platform) steam  what types of diagnostics capabilities are available to us when we need to do production diagnostics? It turns out that a lot of work has been done in this area and specifically .net core 3 promises to bring a wide range of diagnostics capabilities. 
+
+To learn more about production diagnostics in .net core 3, we'll be running through a set of diagnostics scenarios using the built in runtime/sdk tools. The walkthrougs are all run on Ubuntu 16.04 and use the latest .net core preview bits. 
```

Comment:
typo: walkthrougs -> walkthroughs

Reply:
Andrew just heads up, I think we were going to merge this as soon as we got the files in the right spot and then @MarioHewardt was going to keep refining it.